third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -324,7 +324,7 @@ func BootstrapServicePerimeterProjects(t *testing.T, desiredProjects int) []*clo
 	prefixFilter := fmt.Sprintf("id:%s* parent.id:%s", SharedServicePerimeterProjectPrefix, org)
 	res, err := config.clientResourceManager.Projects.List().Filter(prefixFilter).Do()
 	if err != nil {
-		t.Errorf("Error getting shared test projects: %s", err)
+		t.Fatalf("Error getting shared test projects: %s", err)
```

Comment:
I noticed that if the CRM API is not enabled on the test project, the acceptance tests will panic as the reference to `res.Projects` below will be nil. This avoids this happening.